contrib/ansible/roles/master/tasks/packageManagerInstall.yml

```diff
@@ -18,6 +18,10 @@
   when: ansible_distribution == "Fedora" and ansible_distribution_release != "Rawhide"
 
 
+- name: Set fact for Atomic Host package install
+  set_fact:
+    did_install: true
+  when: is_atomic
```

Comment:
Lets put it at the top of file. Pay attention to it so you don't call centOS/fedora install. Then we can rip the is_atomic stuff out of those files.  Does that found good?